com.unity.render-pipelines.high-definition/Editor/Lighting/VolumetricLighting/VolumetricMenuItem.cs

```diff
@@ -6,13 +6,12 @@ namespace UnityEditor.Rendering.HighDefinition
 {
     class VolumetricMenuItems
     {
-        [MenuItem("GameObject/Volume/Density Volume", priority = CoreUtils.Sections.section2 + CoreUtils.Priorities.gameObjectMenuPriority + 2)]
-        static void CreateDensityVolumeGameObject(MenuCommand menuCommand)
+        [MenuItem("GameObject/Local Volumetric Fog", priority = CoreUtils.Priorities.gameObjectMenuPriority + 2)]
```

Comment:
Should it be under Rendering or something? it seems weird to lose the category here

Reply:
Is on the same place as `Decal Projector`, I also think we should grap those two under rendering maybe. What do you think @TheoWong-pixel ?

Reply:
@TheoWong-pixel could you confirm?